exercises/nucleotide-count/canonical-data.json

```diff
@@ -39,12 +63,28 @@
           }
         },
         {
-          "description": "strand with invalid nucleotides",
+          "description": "ignores case in a strand of mixed-case nucleotides.",
           "property": "nucleotideCounts",
-          "strand": "AGXXACT",
+          "strand": "aGcTtTtCaTtCtGaCtGcAaCgGgCaAtAtGtCtCtGtGtGgAtTaAaAaAaGaGtGtCtGaTaGcAgC",
           "expected": {
-            "error": "Invalid nucleotide in strand"
+            "A": 20,
+            "C": 12,
+            "G": 17,
+            "T": 21
           }
+        },
+        {
+          "description": "handles incorrect inputs.",
+          "cases": [
+            {
+              "description": "treats an input with anything else than ACGT-characters as error.",
+              "property": "nucleotideCounts",
+              "strand": "AGUU1+ :'A'}CT",
```

Comment:
I don't think this case is any better or worse than "AGXXACT" at ensuring the description is true. I might even call it worse just for clarity. I would keep AGXXACT. Strength of opinion 2/10.